Add support for SeccompDefault setting for k8s 1.25+ (#3334)
bottlerocket-os/bottlerocket: README.md

* `settings.kubernetes.provider-id`: This sets the unique ID of the instance that an external provider (i.e. cloudprovider) can use to identify a specific node.
* `settings.kubernetes.registry-burst`: The maximum size of bursty pulls.
* `settings.kubernetes.registry-qps`: The registry pull QPS.

> * `settings.kubernetes.seccomp-default`: Exposes the ability to enable RuntimeDefault as the default seccomp profile for all workloads via kubelet-configuration.

Reviewer: ```suggestion
* `settings.kubernetes.seccomp-default`: Enable RuntimeDefault as the default seccomp profile for all workloads via kubelet-configuration. This is disabled by default.
```
